egs/vivos/asr1/RESULTS.md

```diff
@@ -1,128 +1,381 @@
-# pytorch Transducer (enc: VGG2L + 6 x Transformer, dec: 1 x LSTM)
+# RNN-Transducer (enc: 4 x BLSTMP, dec: 1 x LSTM)
```

Comment:
Can you upload the model?
This will help people who only focus on decoding configurations.